ops-tools/docker/images/app/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM node:14-alpine3.11
+
+ADD ./ ./
+WORKDIR /app
+
+RUN node --version && npm --version
+
```

Comment:
You should install NPM dependencies by requirements
RUN npm i

You should do this
EXPOSE 8080